.github/lighthouse/lighthouserc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ci": {
+        "collect": {
+            "settings": {
+                "onlyCategories": ["performance"],
+                "onlyAudits": ["dom-size"]
```

Comment:
Is there an equivalent list we could include like above?

Reply:
Yes as here
https://github.com/GoogleChrome/lighthouse/blob/ed04b10ccfa4e82ee5cd5b437e474a12c1ba52d0/core/config/default-config.js#L178

Reply:
also added in the config